pep-0501.txt

```diff
@@ -501,11 +653,12 @@ a creating a new kind of object for later consumption by interpolation
 functions. Creating a rich descriptive object with a useful default renderer
 made it much easier to support customisation of the semantics of interpolation.
 
-Building atop PEP 498, rather than competing with it
-----------------------------------------------------
+Building atop PEP 701 rather than competing with PEP 498
+--------------------------------------------------------
+
 Earlier versions of this PEP attempted to serve as a complete substitute for
-:pep:`498`, rather than building a more flexible delayed rendering capability on
-top of :pep:`498`'s eager rendering.
+:pep:`498` . With the introduction of 701, this pep can now build a more flexible delayed
```

Comment:
```suggestion
:pep:`498` . With the previous acceptance of :pep:`498` and the more recent introduction of :pep:`701`, this PEP instead now builds a more flexible delayed
```

Reply:
I proposed an [alternate suggestion](https://github.com/python/peps/pull/3047#discussion_r1133057061) that incorporates most of these changes as well as other fixes.

Reply:
Hmm, I need to work out how to tell Github I want to make a multi-line suggestion...

Reply:
Move your mouse to the first/last line you want to select until you see a blue plus sign...

![image](https://user-images.githubusercontent.com/17051931/224484953-701f069f-df33-4834-ac05-36bc129a3b90.png)

Then, click and drag over the lines you want:

![image](https://user-images.githubusercontent.com/17051931/224485012-15c3e788-124f-4242-97bc-493d2b7205c5.png)

Finally, release and boom you've got a multi-line comment

![image](https://user-images.githubusercontent.com/17051931/224485036-2e6f9375-73d2-49be-8672-220683705732.png)

You can also do it by clicking the starting line number and <kbd>Shift</kbd>-clicking the ending line number to select multiple lines, then clicking the plus icon next to the second line number.

See [GitHub's release blog post](https://github.blog/changelog/2020-04-15-multi-line-code-suggestions-general-availability/) and the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) for more details.